src/drt/src/TritonRoute.cpp

```diff
@@ -190,7 +190,20 @@ void TritonRoute::setDebugWorkerParams(int mazeEndIter,
   debug_->markerCost = markerCost;
   debug_->fixedShapeCost = fixedShapeCost;
   debug_->markerDecay = markerDecay;
-  debug_->ripupMode = ripupMode;
+  switch (ripupMode) {
+    case -1:
+      debug_->ripupMode = fr::RipUpMode::UNKNOWN;
```

Comment:
Is it necessary to use the `fr::` namespace here and everywhere else? I think it doesn't need it if it's already in the fr namespace. If it does, I might recommend using an alias for simplicity.